src/main/java/com/google/cloud/tools/appengine/cloudsdk/CloudSdk.java

```diff
@@ -506,7 +506,6 @@ public void validateCloudSdk()
           InvalidJavaSdkException {
```

Comment:
Need to remove `InvalidJavaSdkException`.

Reply:
done